### DISCUSSION TITLE
How can I import a list of users

### DISCUSSION THREAD

**Discussion by andriusign** [directus/directus#5605] [Archive]
I am trying to import a list of users to directus_users. These users are not from Directus system, but from other, I have .csv file which I can modify.
How should I generate ids since they are not being created on insert to database and have to be filled in manually (required field)?

**Comment by rijkvanzanten** [comment]
The IDs are regular UUIDv4 type UUIDs. You can generate these outside of Directus and add them to each user before importing the CSV to the DB 👍🏻
